Tasks/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using ReverseEachWord;
```

Comment:
Do you understand why you used here the using for ReverseEachWord? And do you know what to do not to use it here?

Reply:
Yes i understand why.
Because I do this ReverseWords.ReverseAlphabeticChars.
And not to use it I need to direct inherit like Program : ReverseWords

Reply:
Inheritence is not using for such purposes. We have more suitable tools. Please make it with namespace, like we did